README.md

```diff
@@ -47,30 +49,106 @@ Requirements
 Installation
 ------------
 
-* Checkout or link the code in this repo to
-`/usr/lib/archivematica/automation-tools`
-* Create virtualenv `/usr/share/python/automation-tools` and pip install
-requirements there
-* Create directories `/var/log/archivematica/automation-tools` and
-`/var/archivematica/automation-tools` owned by user `archivematica`, for
-log/database/PID files.
-* Create directory `/etc/archivematica/automation-tools` and add configuration
-files there. Files in the `etc/` directory of this repository can be used as an
-example (also see below for more about configuration)
+Follow each of the steps below to install automation-tools:
+
+1. Make the following directories
+
+```bash
+mkdir -p /usr/lib/archivematica/automation-tools &&
+mkdir /usr/share/python/automation-tools &&
+mkdir -p /var/log/archivematica/automation-tools &&
+mkdir /var/archivematica/automation-tools &&
+mkdir /etc/archivematica/automation-tools
+```
+
+1. Change ownership of the directories to be owned by archivematica (group
+archivematica)
+
+```bash
+chown archivematica:archivematica /var/log/archivematica/automation-tools &&
+chown archivematica:archivematica /var/archivematica/automation-tools
+```
+
+1. Clone this automation-tools repository into your `usr/lib`
+
+```bash
+cd /usr/lib/archivematica/automation-tools/
+git clone https://github.com/artefactual/automation-tools.git .
+```
+
+1. Set up Python's virtual environment
+
+```bash
+cd /usr/share/python/automation-tools
+virtualenv venv
+source venv/bin/activate
+pip install -r /usr/lib/archivematica/automation-tools/requirements.txt
+```
+
+1. (Optional) Update and retrieve required packages
+
+```bash
+apt-get update
+apt-get install p7zip-full
+```
+
+You may also want/need to `apt-get install sudo && apt-get install vim`.
```

Comment:
Today I learned `sudo` itself can be installed like this!! 

Personally, I don't use `vim` but in general, I think here we need to expand on why a user might need to do this. `(For docker users)' might be one approach. 

We could also update the docker images as well. There are a few pieces missing from those components that might be useful to folk.

Reply:
I will leave this out. The terminal will certainly let the user know if they don't have one of these things.